packages/core/src/app/Player.ts

```diff
@@ -278,6 +280,25 @@ export class Player {
     }
   }
 
+  public setAudioVolume(value: number): void {
+    if (value !== this.playerState.current.volume) {
+      this.playerState.current = {
+        ...this.playerState.current,
+        volume: value,
+      };
+    }
+  }
+
+  public addAudioVolume(value: number): void {
+    const newValue = clamp(0, 1, this.playerState.current.volume + value);
+    if (newValue !== this.playerState.current.volume) {
+      this.playerState.current = {
+        ...this.playerState.current,
+        volume: newValue,
+      };
+    }
```

Comment:
Once `setAudioVolume` clamps the value, this could be written as simply:
```suggestion
    this.setAudioVolume(this.playerState.current.volume + value);
```